components/org.wso2.carbon.identity.conditional.auth.functions.user/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/user/Utils.java

```diff
@@ -141,6 +141,7 @@ public static String getUserIdClaimURI(String federatedIdpName, String tenantDom
         if (userNameClaimMapping != null) {
             userIdClaimURI = userNameClaimMapping.getRemoteClaim().getClaimUri();
         }
+        userIdClaimURI = claimConfigs.getUserClaimURI();
```

Comment:
we can move this up right ?

So that we don't need to loop through the claim Mappings